lib/content_location_filter.js

```diff
@@ -29,6 +29,22 @@ function resolvedRedirectResponse(hyper, req, res) {
         // We don't pass the `origin` header further, so possible
         // redirect looping will be handled by normal redirect limiting in `preq`
     })
+    .then((res) => {
+        if (res.status >= 300) {
+            if (redirectNum > 10) {
+                throw new HTTPError({
+                    status: 504,
+                    body: {
+                        type: 'redirect loop',
+                        detail: 'Exceeded maxRedirects'
```

Comment:
maxRedirects written like this suggests there's some config parameter with this name. just write 'max redirects'

Reply:
I have it written that way, since that's what's in the test. Is there a reason that the test was written that way?

Reply:
Regardless, I fixed as you mentioned, including in the test.

Reply:
yeah, fix the test :) this error was originally produced by 'request' library where there's indeed a maxRedirects setting